airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/data/json/JsonSchemaToAirbyteType.kt

```diff
@@ -24,33 +26,33 @@ class JsonSchemaToAirbyteType {
                     "number" -> fromNumber(schema)
                     "array" -> fromArray(schema)
                     "object" -> fromObject(schema)
-                    "null" -> NullType
+                    "null" -> null
```

Comment:
should we treate `{type: null}` schemas as just UnknownType?

Reply:
That makes sense at the field level, but it's still legal(ish?) Airbyte JsonSchema to represent a nullable as `"type": ["null", "string"]`

Reply:
(not blocking, mostly just whinging about jsonschema)

don't we treat everything as nullable anyway? i.e. from our pov, `{type: string} == {type: [null, string]}`

I almost want to have some kind of preprocess somewhere, where we convert all type decls to the array form (`{type: foo}` -> `{type: [foo]}`) and then filter out the `"null"` elements from those arrays

(and then `{type: []}` becomes UnknownType, and we don't need to handle null anywhere)